packages/analytics-browser/src/utils/web-attribution.ts

```diff
@@ -0,0 +1,58 @@
+import { BrowserClient, BrowserConfig } from '@amplitude/analytics-types';
```

Comment:
I see all these functions added. Where is the code being removed from? I see no deletion in our code?

Reply:
Web attribution is a plugin that is released separately. I don't think we should remove it. We only need to remove the web attribution installation.

Reply:
Don't we add the web attribution plugin by default? Or do customers add that manually? Do we have any docs that ask customers to add the web attribution plugin by themselves anywhere?

I am worried that we have duplicate logic in multiple places and its getting harder to figure out which logic is the most accurate. And if we aren't removing the plugin, then the plugin should also use this helper at the very least to avoid code duplication right?

Reply:
What about making the web attribution util part of the WA plugin package and importing it in the client instead of the plugin? 

That way we can support both without duplicating the code

Reply:
The reason I am thinking to put the logic into the plugin so that we don't need to worry about if the plugin is being used in 1.0 or 2.0, if we put the shared logic into the client/core, then the plugin will only work with that version of the Browser SDK.

Reply:
If the WA plugin already has 1.0 and 2.0 versions this may not matter.

Reply:
I think it would be fine, web attribution v1 [depended on ^1.x.x ](https://github.com/amplitude/Amplitude-TypeScript/blob/v1.x/packages/plugin-web-attribution-browser/package.json#L39-L43), [v2 depended on ^2.x.x](https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/plugin-web-attribution-browser/package.json#L39-L43). So there already have breaking change.

Reply:
Then I will leave the web attribution logic in analytics-client-common

Reply:
packages/plugin-web-attribution-browser/src/web-attribution.ts

Is the duplication in this code also being fixed?

Reply:
@izaaz Yes, I updated the web-attribution to import from  analytics-client-common.